guide/src/examples/without-a-bundler.md

```diff
@@ -0,0 +1,27 @@
+# Without a Bundler
+
+[View full source code][code]
+
+[code]: https://github.com/rustwasm/wasm-bindgen/tree/master/examples/without-a-bundler
+
+This example shows how the `--no-modules` flag can be used load code in a
+browser directly (using the same code as the [hello world example][hello]). For
+this deployment strategy bundlers like Webpack are not required. For more
+information on deployment see the [dedicated
+documentation](../reference/deployment.html).
+
+Most of the magic here happens in `index.html`:
+
+```html
+{{#include ../../../examples/without-a-bundler/index.html}}
+```
+
```

Comment:
I'm popping in because my issue (rustwasm/book#150) triggered this change; hope that's okay.

It may be useful to include the lib.rs file too, since the hello world example with modules manually invokes the `greet()` function from Javascript, whereas here it is invoked via `#[wasm_bindgen(start)]`.

Reply:
Agreed.

Reply:
Ah looks like this example has diverged more than I thought from hello world, everything should be updated accordingly now!